activerecord/lib/active_record/associations/preloader/association.rb

```diff
@@ -239,11 +239,11 @@ def build_scope
               scope.merge!(preload_scope)
             end
 
-            if preload_scope && preload_scope.strict_loading_value
-              scope.strict_loading
-            else
-              scope
-            end
+            propagate_strict_loading(scope)
```

Comment:
Even though this is a private method, I'd prefer `cascade` over `propagate` for the name.

Reply:
Renamed to `cascade_strict_loading` as suggested.